frontend/model/contracts/shared/getters/group.js

```diff
@@ -256,9 +256,6 @@ export default ({
     const mincomeCurrency = getters.groupMincomeCurrency
     return mincomeCurrency && currencies[mincomeCurrency]
   },
-  groupMincomeFormatted (state, getters) {
-    return getters.withGroupCurrency?.(getters.groupMincomeAmount)
-  },
```

Comment:
Note that I didn't touch `getters.withGroupCurrency`, which means its output stays deterministic by not getting localized.
As a consequence, notifications emitted as side-effects of `'gi.contracts/group/paymentUpdate'` aren't getting localised.
Not sure how to solve that

Reply:
I see, thanks! In that case, I'll just add that we do already do quite a few things in getters that depend on some global state (e.g., process things differently based on the currently logged in user), although I concur that localisation sounds like something that should be as close as possible to the code responsible for output.

I'll defer it to someone more experienced in Vue to say whether it's fine to leave things as-is or whether defining a new getter is more appropriate. Either way looks fine to me.